### DISCUSSION TITLE
[CLI] "quarto create project" of type "presentation"

### DISCUSSION THREAD

**Discussion by raphaeljolivet** [quarto-dev/quarto-cli#11524] [Feature Requests]
### Description

I just started with Quarto.

I wanted to create a presentation and did : `quarto create project`

*presentation* was not part of the options.

I just created a default project and copy/pasted some starter slides I found around.

The [documentation about presentations](https://quarto.org/docs/presentations/) explains a lot about the syntax but not the starting point : Initial steps to create a presentation and work on it.

It would be nice to be as simple as : 
```bash
> quarto create project -t presentation .
> quarto preview 
```

**Comment by mcanouil** [comment]
Because presentations are not projects per se.
You won't find Typst, HTML, under project.

```
---
format: revealjs
---
```

is all it takes to have a presentation.


> The documentation about presentations explains a lot about the syntax but not the starting point 😀  Initial steps to create a presentation and work on it.


What is missing exactly?
The documentation you linked contain a Quarto document for a presentation.
Nothing more (besides copy/paste) is needed to get started.
Note that VSCode extension and likely RStudio have opinionated commands to create such documents (my own VSCode extension also has such command to create opinionated default documents: https://marketplace.visualstudio.com/items?itemName=mcanouil.quarto-wizard)

```

---
title: "Habits"
author: "John Doe"
format: revealjs
---

## Getting up

- Turn off alarm
- Get out of bed

## Going to sleep

- Get in bed
- Count sheep
```

**Reply by mcanouil** [reply]
Additionally, you can make your own format/template extension to streamline this creation process straight where and how you need it.
